cluster-default-publishing-strategy.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Cluster Publishing Strategy
+authors:
+  - "@abhinavdahiya"
+reviewers:
+  - "@openshift/api-reviewers"
+  - "@openshift/openshift-team-installer"
+  - "@ironcladlou"
+approvers:
+  - "@openshift/api-approvers"
+creation-date: 2019-09-18
+last-updated: 2019-09-18
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Cluster Publishing Strategy
+
+Administrators would like to be able to install OpenShift cluster as `internal/private`, which is only accessible from within the network, VPN tunnels and, not visible to the Internet. Therefore all the cluster operators that create endpoints for the cluster would need a place to discover this intent of the administrator and ensure that their resources are only accessible from within the network, VPN tunnels and not visible to the Internet by default.
+
+This should not block administrators from using operator-level configuration to override publishing strategy of individual endpoints.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+See the [section][#cluster-publishing-strategy]
+
+## Motivation
+
+See the [section][#cluster-publishing-strategy]
+
+### Goals
+
+1. Define an API that allows all the cluster operators to discover the default cluster publishing strategy.
```

Comment:
do defend this goal, you need to list all the operators you believe are affected and which resources they are managing.

Reply:
This was a top level comment before and still outstanding. The only impact I know of is to the default ingress operator and nothing else.